vulture_os/toolkit/api_parser/sentinel_one_mobile/sentinel_one_mobile.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, data):
         self.sentinel_one_mobile_host = data["sentinel_one_mobile_host"]
         if not self.sentinel_one_mobile_host.startswith('https://'):
             self.sentinel_one_mobile_host = f"https://{self.sentinel_one_mobile_host}"
+        if self.sentinel_one_mobile_host.endswith('/'):
+            self.sentinel_one_mobile_host = self.sentinel_one_mobile_host[:-1]
```

Comment:
alternative suggestion
```suggestion
          self.sentinel_one_mobile_host = self.sentinel_one_mobile_host.rstrip('/')
```